dkist/dataset/tests/test_tiled_dataset.py

```diff
@@ -28,6 +29,7 @@ def test_tiled_dataset_slice(simple_tiled_dataset, aslice):
     assert np.all(simple_tiled_dataset[aslice] == simple_tiled_dataset._data[aslice])
 
 
+@pytest.mark.accept_cli_datasets
 @pytest.mark.parametrize("aslice", [np.s_[0, :100, 100:200]])
 def test_tiled_dataset_slice_tiles(large_tiled_dataset, aslice):
```

Comment:
The issue here is that if you pass in a tiled dataset with the `--ds` arg then the tests expecting a dataset object will fail. We could add a `tiled=True` kwarg to `accept_cli_datasets` and a `--tiled-ds` cli arg to differentiate?

Reply:
An alternate thought, instead of `tiled=true` and `fixture=` we could have a `dataset="fixture_name"` and `tiled_dataset="fixture_name"` pair of kwargs?